src/controllers/EvidenceDownloadController.ts

```diff
@@ -25,13 +29,32 @@ export class EvidenceDownloadController extends BaseAsyncHttpController {
 
         const res = this.httpContext.response;
 
-        const metadata = await this.fileTransferService.getFileMetadata(fileId);
+        const metadata: FileMetadata = await this.fileTransferService.getFileMetadata(fileId);
+
         res.setHeader('content-disposition', `attachment; filename=${metadata.name}`);
 
-        const readable = await this.fileTransferService.download(fileId);
+        if (!metadata.av_status || metadata.av_status !== 'clean') {
+            return this.renderDownloadError();
+        }
 
+        let readable = {} as Readable;
```

Comment:
Having `{} as Readable` makes no sense. There is nothing in common between `{}` and `Readable`. Please remove value initiation.